ci-operator/config/maistra/maistra-test-tool/maistra-maistra-test-tool-main__servicemesh-hypershift-4.14-lp-rosa-hypershift.yaml

```diff
@@ -55,21 +59,41 @@ tests:
+              {"step": "servicemesh-kiali-int-tests-execute", "failure_type": "pod_failure", "classification": "Kiali Integration Test Execution", "jira_project": "OSSM", "group": {"name": "lp-tests", "priority": 3}, "jira_component": "Customer Impact", "jira_assignee": "rcernich@redhat.com", "jira_priority": "critical"},
+              {"step": "servicemesh-kiali-int-tests-execute", "failure_type": "test_failure", "classification": "Kiali Integration Test Failure", "jira_project": "OSSM", "group": {"name": "lp-tests", "priority": 3}, "jira_component": "Customer Impact", "jira_assignee": "rcernich@redhat.com", "jira_priority": "critical"},
+              {"step": "servicemesh-maistra-test-tool-execute", "failure_type": "pod_failure", "classification": "Maistra Test Execution", "jira_project": "OSSM", "group": {"name": "lp-tests", "priority": 3}, "jira_component": "Customer Impact", "jira_assignee": "rcernich@redhat.com", "jira_priority": "critical"},
+              {"step": "servicemesh-maistra-test-tool-execute", "failure_type": "test_failure", "classification": "Maistra Test Failure", "jira_project": "OSSM", "group": {"name": "lp-tests", "priority": 3}, "jira_component": "Customer Impact", "jira_assignee": "rcernich@redhat.com", "jira_priority": "critical"}
             ]
         }
       FIREWATCH_DEFAULT_JIRA_ADDITIONAL_LABELS: '["4.14-lp","rosa-hypershift-lp","servicemesh-lp"]'
+      FIREWATCH_DEFAULT_JIRA_ASSIGNEE: rcernich@redhat.com
+      FIREWATCH_DEFAULT_JIRA_COMPONENT: '["Customer Impact"]'
+      FIREWATCH_DEFAULT_JIRA_PRIORITY: critical
       FIREWATCH_DEFAULT_JIRA_PROJECT: LPINTEROP
+      FIREWATCH_FAIL_WITH_TEST_FAILURES: "true"
+      GATEWAY_API_ENABLED: "true"
       OCM_LOGIN_ENV: staging
       OPENSHIFT_VERSION: "4.14"
       OPERATORS: |
         [
+            {"name": "elasticsearch-operator", "source": "redhat-operators", "channel": "stable", "install_namespace": "openshift-operators"},
+            {"name": "jaeger-product", "source": "redhat-operators", "channel": "stable", "install_namespace": "openshift-operators"},
+            {"name": "kiali-ossm", "source": "redhat-operators", "channel": "stable", "install_namespace": "openshift-operators"},
             {"name": "servicemeshoperator", "source": "redhat-operators", "channel": "stable", "install_namespace": "openshift-operators"}
         ]
       REGION: us-west-2
+      SMCP_VERSION: "2.4"
+      TEST_GROUP: interop
     test:
     - ref: install-operators
     - ref: cucushift-hypershift-extended-health-check
+    - ref: servicemesh-maistra-test-tool-execute
+    - ref: servicemesh-smcp-install
+    - ref: servicemesh-kiali-int-tests-execute
     workflow: firewatch-rosa-aws-sts-hypershift
```

Comment:
is that `firewatch-rosa-aws-sts-hypershift` workflow implemented by another team ?
where is your cluster provisioning step ? or are you login to an existing cluster ?

Reply:
yes firewatch is added for INTEROP testing